docs/reference/api/docker_remote_api_v1.20.md

```diff
@@ -1061,6 +1063,112 @@ Status Codes:
 -   **404** – no such container
 -   **500** – server error
 
+### Get an archive of a filesytem resource in a container
+
+`GET /containers/(id)/archive`
+
+Get an tar archive of a resource in the filesystem of container `id`.
+
+Query Parameters:
+
+- **path** - resource in the container's filesystem to archive. Required.
+
+    If not an absolute path, it is relative to the container's root directory.
+    The resource specified by **path** must exist. To assert that the resource
+    is expected to be a directory, **path** should end in `/` or  `/.`
+    (assuming a path separator of `/`). If **path** ends in `/.` then this
+    indicates that only the contents of the **path** directory should be
+    copied. A symlink is always resolved to its target.
+
+    > It is not possible to copy certain system files such as resources under
+    > `/proc`, `/sys`, `/dev`, and mounts created by the user in the container.
+
```

Comment:
No, you can. 'mounts created by the user in the container' includes things like FUSE mounts made by the process(es) in the container or anything added to fstab after the container is created.